test/integration/scheduler_perf/test-performance.sh

```diff
@@ -43,10 +45,10 @@ if ${RUN_BENCHMARK:-false}; then
   go test -c -o "perf.test"
 
   kube::log::status "performance test (benchmark) start"
-  "./perf.test" -test.bench=. -test.run=xxxx -test.cpuprofile=prof.out -test.short=false
+  "./perf.test" $glog_flags -test.bench=. -test.run=xxxx -test.cpuprofile=prof.out -test.short=false
   kube::log::status "...benchmark tests finished"
 fi
 # Running density tests. It might take a long time.
 kube::log::status "performance test (density) start"
-go test -test.run=. -test.timeout=60m -test.short=false
+go test $glog_flags -test.run=. -test.timeout=60m -test.short=false
```

Comment:
this actually shouldn't even work, because `-v` is a boolean flag to `go test`, https://github.com/kubernetes/kubernetes/pull/79242 does not have this problem

Reply:
The result of $glog_flags only takes the form of `-v=3`, not `-v 3`, so there is no conflict.

Reply:
This is not resolved:

``
$ go test -v=3 -test.run=. -test.timeout=60m -test.short=false

go test: illegal bool flag value 3
run "go help test" or "go help testflag" for more information
```